starfish/multiprocessing/pool.py

```diff
@@ -0,0 +1,26 @@
+import multiprocessing.pool as mp
+from typing import Optional
+
+
+class Pool(mp.Pool):
+    """Wrapper class for multiprocessing pool. If n_processes=1 just use map on main
+    thread for debugging purposes """
+
+    def __init__(self, processes: Optional[int]=None, **kwargs):
```

Comment:
To make this work exactly as multiprocessing.Pool, you would probably need:

`def __init__(self, processes: Optional[int]=None, *args, **kwargs):`

and then pass *args into mp.Pool's constructor.